tsconfig.cjs.json

```diff
@@ -1,12 +1,20 @@
 {
   "compilerOptions": {    
+    "strict": true,
```

Comment:
You should define it in `tsconfig.esm.json` too, shouldn't you?

Reply:
AFAIK, this setting will be inherited because [*tsconfig.esm.json* extends *tsconfig.cjs.json*](https://github.com/configcat/common-js/blob/676de9c02ab715d1e27b99ae1d0643ee4d17882f/tsconfig.esm.json#L2).

> The configuration from the base file are loaded first, then overridden by those in the inheriting config file. All relative paths found in the configuration file will be resolved relative to the configuration file they originated in.
> [...]
> Currently, the only top-level property that is excluded from inheritance is [references](https://www.typescriptlang.org/tsconfig#references).

(https://www.typescriptlang.org/tsconfig#extends)

As a matter of fact, it'd be nice to revise the current tsconfig hierarchy because we have no *tsconfig.json* file in this repo, which is problematic for multiple reasons. E.g. VSCode needs that file to display correct diagnostics. But let's address this in another PR to avoid merge conflicts.

Reply:
You are right, I ought to known my lines better :P